src/main/java/hello/postpractice/advice/exception/EmailNotFailedCException.java

```diff
@@ -0,0 +1,13 @@
+package hello.postpractice.advice.exception;
+
+public class EmailNotFailedCException extends RuntimeException {
+    public EmailNotFailedCException() {
+        super();
+    }
+    public EmailNotFailedCException(String message) {
```

Comment:
이런식으로 하위에 뭔가를 받아서 에러를 만드려면 상위 에러를 정의하고 (공통 익셉션 모델)
상위 에러에서 상속받는 에러가 특수한 메시지를 넣을 수 있게 만드는 것이 좋을 듯함

```suggestion
~~~ extends CommonException {
public EmailNotFailedCException(String message, Throwable cause) {
        super(message, cause);
    }
```